### PR TITLE
updated pub-global.md to new `dart` tool

### PR DESCRIPTION
**Pull request by Abhishek01039**
Added dart pub global to this page `src/tools/pub/cmd/pub-global.md`

### CODE REVIEW

**Comment by Abhishek01039** [comment]
@kwalrath 
can you please review this PR?
Thanks

File: src/tools/pub/cmd/pub-global.md [hunk 1]
```diff
@@ -17,20 +17,20 @@ For example, say you want to run
...
+$ dart pub global activate stagehand
 $ stagehand
 ```
 
 If this doesn't work, you might need to
 [set up your path](#running-a-script-from-your-path).
 
 To run a Dart script from within a package, or from a
-package that your package depends on, see [pub run](/tools/pub/cmd/pub-run).
+package that your package depends on, see [dart run](/tools/pub/cmd/pub-run).
```
Comment (kwalrath) [inline src/tools/pub/cmd/pub-global.md:28]: The URL needs to be updated. For now, let's use /tools/dart-tool.

**Review by kwalrath — COMMENTED** [review]
LGTM aside from one URL

File: src/tools/pub/cmd/pub-global.md (hunk 1 above)
Comment (Abhishek01039) [reply src/tools/pub/cmd/pub-global.md]: Ah ok @kwalrath 
can you please review it again?
Thanks

**Review by kwalrath — APPROVED** [review]
LGTM. Thanks, @Abhishek01039!

**Comment by Abhishek01039** [comment]
Thanks @kwalrath

**Pull request merged**